run_dir/design/base_b5.html

```diff
@@ -28,7 +28,11 @@
 
 </head>
 
+{% try %}
+<body {{ body_attrs }}>
```

Comment:
Hmm...

Reply:
Yes, that one might call for an explanation. I had to put some custom attributes on the body to get the scrollspy to work on the full page. But I didn't want to put it there for all the pages. So now there is the possibility to add a `body_attrs` for any page and it will be there. The try should make it work for all pages without the variable as well.